docs/workflow/blocks/link_action_block.rst

```diff
@@ -0,0 +1,101 @@
+Link Action block
+===========
+The Link Action block provides a user-friendly button that links to other Flows. 
+Like a normal link, the user can choose how Flows are opened, according to their browser configuration.
+
+Existing Flows that use a combination of an :doc:`Actions block <actions>` and a :doc:`Launch Block <launch_block>` to open a Flow, can be replaced by the Link Action block to provide the user with more control.
+
+What problem does it solve?
```

Comment:
I am not sure of this bit of text.
We do not normally write an explanation of which problem the block solves, and I do not think it is necessary. 
I like the idea of having more explanation of what a block does and where it can be used, but not which problem it solved.
Most of this is a general explanation of how a link works in a browser, rather than the Link Block in Kendraio.

If you want to add more content to the description (at Line 7), I think you have some material in this text that can be useful, like: to use right click or to see the link address in the corner of the browser windows. 
But I wouldn't add more than this.

Reply:
Thanks, I made it shorter explanation instead. @gsambrotta

Reply:
The new explanation looks great! Thank you for following the suggestion